src/LogicTasks/Semantics/Decide.hs

```diff
@@ -110,6 +110,41 @@ start = []
 
 partialGrade :: OutputMonad m =>  DecideInst -> [Int] -> LangM m
 partialGrade DecideInst{..} sol = do
+  preventWithHint (null sol)
+    (translate $ do
+      german "Lösung enthält Indizes?"
+      english "Solution contains indices?"
+    )
+    (translate $ do
+      german "Die Lösung beinhaltet mindestens einen Index."
+      english "The solution contains at least one index."
+    )
+
+  preventWithHint (solLen > tableLen)
+    (translate $ do
+      german "Lösung überschreitet nicht maximale Anzahl Werte?"
+      english "Solution does not exceed maximum possible number of values?"
```

Comment:
Habe mir angewöhnt, in den Hinweisen an Studierende in solchen Situationen lieber "count of" statt "number of" zu schreiben. Es kommen sonst mit gewisser Wahrscheinlichkeit (die bei großer Anzahl Studierender zu einer absoluten Zahl wird) Verwirrungen auf wie "das meint, was die größte Zahl in der Liste sein soll".